### ISSUE TITLE
LCP passphrase input HTML element should be automatically focused when dialog pops-up

### ISSUE THREAD

**Issue by danielweck** [edrlab/readium-desktop#80]
...otherwise, typing requires extra user interaction to set the focus, and also TAB-navigation occurs "behind" the dialog because of lack of proper modal support

**Comment by danielweck** [comment]
Use React `Ref`? https://reactjs.org/docs/refs-and-the-dom.html
